hello-strongbox-yarn/README.md

```diff
@@ -0,0 +1,3 @@
+This is an example project illustrating how to use the Strongbox artifact repository manager with YARN.
```

Comment:
Cosmetic: I think there's no need for Yarn to be in capital letters. It's not an abbreviation, is it? It think it would be fine like this:
```
This is an example project illustrating how to use the Strongbox artifact repository manager with Yarn.
```